dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESContentFactoryImpl.java

```diff
@@ -2157,6 +2157,10 @@ private static String findAndReplaceQueryDates(String query) {
                 query = query.replaceAll( " to ", " TO " );
             }
 
+            if (query.contains( " or " )) {
+                query = query.replaceAll( " or ", " OR " );
```

Comment:
Are we sure that the String " or " will not be replaced in other parts of the query where it IS supposed to be lower-case?

Reply:
@jcastro-dotcms I was repeating the same fix we did above for the `TO`. But I think it's a valid concern

Reply:
@jcastro-dotcms we re-worked this based on your valid concerns